lib/rubocop/cop/lint/redundant_string_coercion.rb

```diff
@@ -3,26 +3,33 @@
 module RuboCop
   module Cop
     module Lint
-      # Checks for string conversion in string interpolation,
+      # Checks for string conversion in string interpolation, `print`, `puts`, and `warn` arguments,
       # which is redundant.
       #
       # @example
       #
       #   # bad
       #
       #   "result is #{something.to_s}"
+      #   print something.to_s
+      #   puts something.to_s
```

Comment:
Seems you forgot `warn something.to_s` here.

Reply:
Oops, I've added it.